### PR TITLE
Add skip_builds_for_existing_commits

### PR DESCRIPTION
**Pull request by jradtilbrook**
This adds docs for the `skip_builds_for_existing_commits` provider settings

### CODE REVIEW

**Comment by jradtilbrook** [comment]
@mbelton-buildkite I tried to run this locally with `docker-compose up --build` but I got an error page. Can we catch up next week to see what the problem is?

File: pages/apis/rest_api/pipelines.md [hunk 1]
```diff
@@ -1111,6 +1111,12 @@ Properties available for Bitbucket Cloud, GitHub, and GitHub Enterprise:
       <p class="Docs__api-param-eg"><em>Example:</em> <code>"features/*"</code></p>
     </td>
   </tr>
+  <tr>
+    <th><code>skip_builds_for_existing_commits</code></th>
+    <td>
+      Whether to skip creating a new build if an existing build for the commit and branch already exists.
```
Comment (mbelton-buildkite) [inline pages/apis/rest_api/pipelines.md:1117]: Avoid saying it exists twice:

```suggestion
      Whether to skip creating a new build if a build for the commit and branch already exists.
```

**Review by mbelton-buildkite — APPROVED** [review]
Sorry @jradtilbrook, I missed this one! I've left a suggestion for a slight tweak, but looks good in general.

**Comment by mbelton-buildkite** [comment]
> @mbelton-buildkite I tried to run this locally with `docker-compose up --build` but I got an error page. Can we catch up next week to see what the problem is?

I've stopped running the site locally with Docker because I was running into performance issues. Is that workflow still supported @dannymidnight? Instead, I've been using the `foreman` command.

**Comment by mbelton-buildkite** [comment]
^ merged latest main to pick up URL changes that should fix the broken build

**Review by mbelton-buildkite — APPROVED** [review]
Looks good and builds passing. Feel free to merge when you're ready :)

**Pull request merged**